_releases/2022-10-06-1.6.0-released.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Crystal 1.6.0 is released!
+version: 1.6.0
+summary:
+thumbnail: +
+author:
+---
+
+We are delivering a new release with several bugfixes and improvements. Below we list the most important or interesting changes, without mentioning several bugfixes and smaller enhancements. For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.6.0). Breaking changes are marked with ⚠️.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.6.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [183 changes since the 1.5.1 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.6.0) by 26 contributors. We thank all the effort put into improving the language! ❤️
+
+Below we list the most remarkable changes in the language and stdlib.
+
+## ⚠️ Refactor in console methods
+
+By making some console methods available in Windows, a significant [refactor](https://github.com/crystal-lang/crystal/pull/12352) lead to deprecating three macro methods in `FileDescriptor`: `cooked_from_tc_mode!`, `noecho_from_tc_mode!`, and `raw_from_tc_mode!`. Additionally, methods `#noecho!` and `#raw!` now return `nil`.
+
+## ⚠️ Refactors in `File::Info`
+
+[Another refactor](https://github.com/crystal-lang/crystal/pull/12385) lead to an improvement in the `File` API, which was breaking the abstraction by returning an object intended to be internal. In practice, if for some reason you ended up with an object of type `Crystal::System::FileInfo`, it should now be `File::Info`.
```

Comment:
Ditto. This is also a very irrelevant change for users. It should not be on the top.